uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/leave-request-popup-controller_test.js

```diff
@@ -554,6 +577,105 @@
           });
         });
       });
+
+      describe('when absence period is changed', function () {
+        describe('for multiple days', function () {
+          describe('before from date is selected', function () {
+            it('should disable to date and to type', function () {
```

Comment:
the to date and type are dependent if `$ctrl.leaveRequest.from_date` is set or not and hence the check